packages/commons/src/logging/loggerMiddleware.ts

```diff
@@ -129,20 +96,37 @@ const getLogger = (serviceName?: string) =>
     silent: process.env.NODE_ENV === "test",
   });
 
-export const loggerMiddleware = (serviceName: string) => () =>
-  expressWinston.logger({
-    winstonInstance: getLogger(serviceName),
-    requestWhitelist:
-      config.logLevel === "info" ? ["body", "headers", "query"] : [],
-    ignoredRoutes: ["/status"],
-    responseWhitelist:
-      config.logLevel === "info" ? ["body", "statusCode", "statusMessage"] : [],
-    meta: false,
-    msg: (req, res) =>
-      `Request ${req.method} ${req.url} - Response ${res.statusCode} ${res.statusMessage}`,
-  });
+// export const loggerMiddleware = (serviceName: string) => () =>
+//   expressWinston.logger({
+//     winstonInstance: getLogger(serviceName),
+//     requestWhitelist:
+//       config.logLevel === "info" ? ["body", "headers", "query"] : [],
+//     ignoredRoutes: ["/status"],
+//     responseWhitelist:
+//       config.logLevel === "info" ? ["body", "statusCode", "statusMessage"] : [],
+//     meta: false,
+//     msg: (req, res) =>
+//       `Request ${req.method} ${req.url} - Response ${res.statusCode} ${res.statusMessage}`,
+//   });
```

Comment:
Does removing this middleware mean that request/response info (method, URL, status code, etc.) is not logged anymore? Have you already explored alternative solutions? I think logging this data is a requirement that we must satisfy somehow

Reply:
https://github.com/pagopa/interop-be-monorepo/pull/434